osi_object.proto

```diff
@@ -359,6 +392,29 @@ message MovingObject
     //
     optional MovingObjectClassification moving_object_classification = 9;    
 
+    // External Reference to the MovingObject Source
```

Comment:
ßWe should point out, that this is not mandatory, but optional. Also at the lother places, what do you think